conjure_oxide/tests/integration/basic/abs/04-bounds/input.eprime

```diff
@@ -0,0 +1,7 @@
+language ESSENCE' 1.0
+
+$ should get int(0..3), not int(2..3)
+find x : int(-2..3) 
+
+such that 
+|x| = 1 
```

Comment:
ok, so this is not enough since |x| is not extracted into an aux var, of course.

would something like `y * |x| = 2` with `find y : int(1..3)` create an aux for `|x|`?

Reply:
Works 👍🏻